point-of-sale-app/skaffold.yaml

```diff
@@ -23,13 +23,31 @@ deploy:
     - k8-manifests/service-configs.yaml
 # [END anthosbaremetal_point_of_sale_app_skaffold_config_setup]
 ---
+# [START anthosbaremetal_point_of_sale_app_skaffold_config_db]
+apiVersion: skaffold/v2beta21
+kind: Config
+metadata:
+  name: db # module defining spring profile & db configuration
+deploy:
+  kubectl:
+    manifests:
+    - k8-manifests/springprofile-mysql.yaml
+    - k8-manifests/mysql-db.yaml
+profiles:
+- name: inmemory
```

Comment:
I'm a bit confused about the inmemory profile -- in this skaffold config, my understanding is that `mysql-db` gets deployed, but Spring is configured with an inmemory profile. Isn't this wasteful to deploy mysql if it doesn't get used? Am I reading this config wrong?

Reply:
Here if the profile is activated then the `default` configs will not get deployed. Only whats listed under the `inmemory` gets deployed. The public skaffold docs on this is a bit confusing. I raised this with the skaffold team.